### DISCUSSION TITLE
add register User page

### DISCUSSION THREAD

**Discussion by daisy-yangyang** [hoppscotch/hoppscotch#1743] [Ideas]
The idea is to 1. Add registered user functionality 2. Save the Collections to the computer which deployed Hoppscotch and associate with the user 3.  After logging in, the Collections saved by the user is displayed

**Comment by liyasthomas** [comment]
Users can log in to Hoppscotch with their Google, GitHub, or email and sync collections, environments, history, and customised settings.

Click on the "Login with" icon button to sync your local collections and all activity to the cloud. Access them from any other devices any time just by signing in again.

<img width="1440" alt="Screenshot 2021-07-19 at 3 11 33 PM" src="https://user-images.githubusercontent.com/10395817/126140412-67852920-5573-40cb-b0ed-0a272804e5a2.png">

**Reply by MeBetterMan** [reply]
if in offline env. how can i log in? 
image the scene : i deploy Hoppscotch in my local area network ,which is not accessed to the Internet. How can my team to
log in to Hoppscotch to sync collections, environments, history, and customised settings？

Click on the "Login with" icon button to sy

**Reply by liyasthomas** [reply]
You will need an active internet connection to make use of cloud sync etc.

The offline sync feature is not in the roadmap.

**Reply by MeBetterMan** [reply]
a good and useful feature 。have a good consideration pls。
